### ISSUE TITLE
lib/index.js in ES5

### ISSUE THREAD

**Issue by fvicente** [nomiddlename/date-format#5]
Hi! thanks for this useful library.
I'm using it in my project, but unfortunately, I cannot use it directly from the node_modules, instead I had to copy the source to my project tree (nothing too bad, your library is tiny after all!). The reason for that is that I'm using webpack which uses uglifyjs for 'production' builds, and uglifyjs doesn't recognises ES6, only ES5. I get the following error:

```
ERROR in bundle.js from UglifyJs
SyntaxError: Unexpected token: name (numAsString) [./~/date-format/lib/index.js:4,0]
```
I use ES6 like you, but the difference is that my code gets translated to ES5 with `babel` before the uglify.

Some [people on the Internet suggests](http://krasimirtsonev.com/blog/article/javascript-library-starter-using-webpack-es6) that you can develop your libraries in ES6 and release it in ES5 using babel, then you will allow a wider audience for date-format.
Your library's source tree would look like this.

````
+-- lib
|   +-- library.js (es5)
|   +-- library.min.js (es5)
+-- src
    +-- index.js (es6)
    +-- helpers.js (es6)
````

What do you think about this? are you interested at all in adding this support for ES5 or just 'not your problem'? 😄

**Comment by nomiddlename** [comment]
I'm on holiday at the moment, so I won't get to check this out properly for a week or two. But my first thought is that there isn't much code in the library, so changing it to be ES5 compatible would probably be easier than adding babel and stuff.

**Comment by makedonec88** [comment]
Hey, have the same issue.
I've temporary fixed it in `webpack.config.js` by this code:
```javascript
{
  test: /\.js$/,
  loader: 'babel-loader',
  exclude:  /node_modules\/(?!(date-format)\/).*/, 
  query: {
    presets: ['es2015']
  }
},
```
Where, in `exclude` we're excluding everything from `node_modules`, except `date-format`. More information [here](https://github.com/webpack/webpack/issues/2031#issuecomment-219040479).

Although, I believe you can do the same in `.babelrc`, if you're using babel with something else, than webpack.

**Comment by nomiddlename** [comment]
I've reverted the code to use ES5 - it should behave nicely for you now. Published to npm in version 1.1.0
